packages/cli/src/cmds/update.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Result, FxError, err, ok } from "@microsoft/teamsfx-api";
+import path from "path";
+import { Argv } from "yargs";
+import activate from "../activate";
+import CliTelemetry, { makeEnvRelatedProperty } from "../telemetry/cliTelemetry";
+import {
+  TelemetryEvent,
+  TelemetryProperty,
+  TelemetrySuccess,
+} from "../telemetry/cliTelemetryEvents";
+import { getSystemInputs } from "../utils";
+import { YargsCommand } from "../yargsCommand";
+
+export class UpdateAadApp extends YargsCommand {
+  public readonly commandHead = "aad-app";
+  public readonly command = this.commandHead;
+  public readonly description = "Update the Aad App Manifest in the current application.";
```

Comment:
@jidddddd please help review this Description

Reply:
LGTM